frontend/src/components/UserInfo/UserAvatar.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import {withStyles} from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+
+import CardMedia from '@material-ui/core/CardMedia';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import CardContent from '@material-ui/core/CardContent';
+import Typography from '@material-ui/core/Typography';
+
+const styles = {
+    card: {
+        flexGrow: 1,
+        maxWidth: 345,
+        width: "50%",
+        float: "left",
+        margin: 16,
+
```

Comment:
You don't really need `float` property almost in all cases, however your `float` isn't work here because your element inside `flex` container
Please for positioning  your elements use `<Grid/>` component or at least `flex-box` model

Reply:
Thanks for info. I will use Grid.